dashboard/app/controllers/openai_sessions_controller.rb

```diff
@@ -0,0 +1,44 @@
+class OpenaiSessionsController < ApplicationController
```

Comment:
@bencodeorg @molly-moen For my high-level knowledge, I have two questions:
1. Can someone clarify why this warrants being called a sessions controller? I think of sessions as being related to user authentication, authorization, and maintaining user-specific data across multiple requests... other than authorizing the resource here and checking that the parameters are required, this doesn't seem to be dealing with "sessions"? 
2. What is the logical division between the "SessionsController" and the "Helper" here? If the helper is responsible for pulling the relevant data off the response, shouldn't it be responsible for setting the headers, too?

Reply:
I guess I'm just trying to call out that the organization here feels unintuitive to me, and want to make sure I'm learning about what the ideal is 😄

Reply:
Good qs -- re: #1, I didn't pick up the file name change from the first iteration (it was OpenaiController I believe?). I think you're right that sessions doesn't necessarily mean much in this context, and was probably copied over from the javabuilder example.

Also agree with your second point that moving some of the header stuff into the helper would make sense!

Reply:
Thanks Emily! Yes - definitely makes sense to move the setting of headers to the helper file - thanks!
The logical division for me was that the helper defined functions that directly interacted with the Open AI API - it also makes it easier to stub the code we don't directly control for testing purposes.

Reply:
In terms of sessions in the name, I think it does make sense to remove based on your feedback. I was just using `javabuilder_sessions_controller` as an example since I'm new to this area of code base. Thanks!